core/block_svg.js

```diff
@@ -634,11 +634,12 @@ Blockly.BlockSvg.prototype.showHelp_ = function() {
 };
 
 /**
- * Show the context menu for this block.
- * @param {!Event} e Mouse event.
+ * Generate the context menu for this block.
  * @private
```

Comment:
This should be protected instead of private since it's intended to be overwritten.

Reply:
Hrm. I'm actually not sure if protected method standard is to have an underscore or not... is this what you were expecting?

Reply:
We pretty consistently leave the underscore on there for protected methods in Blockly. I'm not sure what the wider style norms are, though.

Reply:
Sorry to butt in, but that's not quite true.

If a protected method has an underscore in Blockly, it means it used to be marked private but was obviously part of our API, so we changed the annotation without changing the name.

New protected methods have no underscore.

Reply:
Thanks for the correction, Rachel. =) Merging as is, then.